pkg/constants/constants.go

```diff
@@ -37,6 +37,7 @@ const (
 	// TokenValidator is used to name the resources used to allow the kube-apiserver to validate tokens against the oidc authenticator.
 	TokenValidator = ApplicationName + "-token-validator"
 	// TokenValidatorDir is the path were the token-validator related secrets are mounted in the kube-apiserver pod.
+	/* #nosec */
```

Comment:
Can we have this as a line comment after the constant declaration?

Reply:
We can have ```//nolint:gosec``` on the same line after the constant declaration